httpserver/rate_limiter.cpp

```diff
@@ -0,0 +1,47 @@
+#include "rate_limiter.h"
+
+#include <algorithm>
+
+constexpr static uint64_t FILL_EMPTY_BUCKET_MS =
```

Comment:
Consider using microseconds instead? Also use the data type from `std::chrono`?

Reply:
Done